src/aitsi/m3spin/commons/impl/AssignmentImpl.java

```diff
@@ -1,12 +1,24 @@
 package aitsi.m3spin.commons.impl;
 
+import aitsi.m3spin.commons.enums.EntityType;
 import aitsi.m3spin.commons.interfaces.Assignment;
 import aitsi.m3spin.commons.interfaces.Expression;
 import aitsi.m3spin.commons.interfaces.Variable;
 import lombok.AllArgsConstructor;
+import lombok.EqualsAndHashCode;
+import lombok.NoArgsConstructor;
 
 @AllArgsConstructor
-public class AssignmentImpl implements Assignment {
+@NoArgsConstructor
+@EqualsAndHashCode(callSuper = true)
+public class AssignmentImpl extends TNodeImpl implements Assignment {
+    private static final EntityType TYPE = EntityType.ASSIGNMENT;
+
     private Variable var;
```

Comment:
`var` jest w sumie słowem kluczowym w Javie, może lepiej nie używać?

Reply:
Chyba dopiero od 10, ale to w sumie kwestia do uzgodnienia

Reply:
A my używamy 8 czy jak? Bo ja chyba w IntelliJ ustawiłem 11 XD

Reply:
Tak, używamy 8 XD

Reply:
W takim razie nie było tematu XD